server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -106,6 +106,12 @@ def _check_parameters(self, docs):
                     .format(doc['name'], doc['operator']))
 
     def _check_equals(self, docs):
+        """
+        Checks if any of the filter conditions in the docs
+        already exists
+        :param docs: List of filter conditions to be tested
+        :return: Raises an exception if it exists
```

Comment:
In Sphinx the `:raises ExcType:` directive should be used for describing exceptions potentially raised by a method.